### DISCUSSION TITLE
shokax更新之后无法渲染

### DISCUSSION THREAD

**Discussion by FidStyle** [theme-shoka-x/hexo-theme-shokaX#217] [General]
使用npm更新之后无法渲染，应该是player的问题（）

![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/1464a7c8-98f4-4440-ace4-fbbbdf5f6a29)

<img width="1203" alt="image" src="https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/bfe5cc0f-3656-492d-98f6-a1b2438fe8d6">

**Comment by FidStyle** [comment]
@zkz098

**Reply by zkz098** [reply]
是player的问题
等0.4发布，0.4重构了ts打包机制，彻底修复了这个问题

**Comment by FidStyle** [comment]
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/72a8fbb9-916a-4cda-911c-fc55cadb13d8)
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/027fde49-17e2-4261-8883-54dbe1041dc5)
对了下源码，发现0.3.13做了如上更改，导致player未设置为false时会渲染错误
是否有必要release一个0.3.x做下修复（）
@zkz098

**Reply by FidStyle** [reply]
/* @__PURE__ */ 惹的祸（）

**Reply by zkz098** [reply]
0.4正式版出了之后0.3就EOL了，0.4也没删功能应该能无损迁移
